deploy/k8s/gprofiler.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: granulate
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: granulate-gprofiler
+  namespace: granulate
+  labels:
+    app: granulate-gprofiler
+spec:
+  selector:
+    matchLabels:
+      app: granulate-gprofiler
+  template:
+    metadata:
+      labels:
+        app: granulate-gprofiler
+    spec:
+      hostPID: true
+      securityContext:
+        runAsUser: 0
+        runAsGroup: 0
+      restartPolicy: Always
+      containers:
+        - name: gprofiler
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              cpu: 100m
+              memory: 256m
+            limits:
+              cpu: 512m
```

Comment:
The cpu value of 512m is not make sense here, 1000m is one core so 500m is a half core (not powers of 2).
https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/#cpu-units

Reply:
You are absolutely right